.github/workflows/release.yml

```diff
@@ -50,13 +35,6 @@ jobs:
       - name: Run tests
         run: yarn test && yarn spec:e2e
 
-      - name: Release --dry-run  # Uses release.config.js
```

Comment:
From my understanding, if we remove this there won't be anything published. My expectation was that we would npm auto publish and create new semver based on commit type.

Reply:
This step used [`--dry-run`](https://semantic-release.gitbook.io/semantic-release/usage/configuration#dryrun) when user opted in for this option during the manual trigger. Since we're auto-running on all commits to master it's not necessary.

The real npm autopublish command [still exists here](https://github.com/amilajack/eslint-plugin-compat/pull/444/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34L60)